pertpy/plot/_mixscape.py

```diff
@@ -58,7 +58,7 @@ def barplot(  # pragma: no cover
 
         Args:
             adata: The annotated data object.
-            control: Control category from the `pert_key` column. Default is 'NT'.
+            guide_rna_column: The column of `.obs` with guide RNA labels. The target gene labels. The format should be <gene_target>g<#>. For example, 'STAT2g1' and 'ATF2g1'.
```

Comment:
```suggestion
            guide_rna_column: The column of `.obs` with guide RNA labels. The target gene labels. The format must be <gene_target>g<#>. For example, 'STAT2g1' and 'ATF2g1'.
```

Is there any way to make this a bit more flexible? Why do our requirements have to be so strict?